frontend/src/metabase/components/AddToDashSelectDashModal.jsx

```diff
@@ -62,6 +63,13 @@ export default class AddToDashSelectDashModal extends Component {
                         items={this.state.dashboards}
                         onClickItemFn={this.addToDashboard}
                     />
+                    <button
+                        className="Button Button--primary"
+                        onClick={() => this.setState({shouldCreateDashboard: true})}
```

Comment:
Minor nitpick, we generally put spaces inside the brackets here, e.x. `{ shouldCreateDashboard: true }`